Dockerfile

```diff
@@ -8,7 +8,7 @@
 # dockerfile to spin up a container suitable for generation of documentation
 
 FROM centos:7
-LABEL maintainer="toolkit@shotgunsoftware.com"
+LABEL maintainer="https://shotgridsoftware.com/"
```

Comment:
Is this information we can see on the generated doc? Does that work?

Reply:
This only adds metadata to the Docker image according to [official docs](https://docs.docker.com/engine/reference/builder/#label).